cfn-resources/serverless-instance/cmd/resource/resource.go

```diff
@@ -0,0 +1,395 @@
+package resource
+
+import (
+	"context"
+	"fmt"
+	"strings"
+
+	"github.com/aws-cloudformation/cloudformation-cli-go-plugin/cfn/handler"
+	"github.com/aws/aws-sdk-go/service/cloudformation"
+	"github.com/mongodb/mongodbatlas-cloudformation-resources/util"
+	"github.com/mongodb/mongodbatlas-cloudformation-resources/util/constants"
+	log "github.com/mongodb/mongodbatlas-cloudformation-resources/util/logger"
+	progress_events "github.com/mongodb/mongodbatlas-cloudformation-resources/util/progressevent"
+	"github.com/mongodb/mongodbatlas-cloudformation-resources/util/validator"
+	mongodbatlas "go.mongodb.org/atlas/mongodbatlas"
+)
+
+const (
+	CallBackSeconds = 30
+)
+
+var CreateRequiredFields = []string{constants.ProjID, constants.PvtKey, constants.PubKey, constants.Name, constants.Name}
```

Comment:
```suggestion
var CreateRequiredFields = []string{constants.ProjID, constants.PvtKey, constants.PubKey, constants.Name}
```

Reply:
no other properties are required for the creation?

Reply:
No, Just the Name is required param